docs/reference/components/transforms/route.cue

```diff
@@ -36,7 +36,7 @@ components: transforms: swimlanes: {
 
 	configuration: {
 		lanes: {
-			description: "A table of swimlane identifiers to logical conditions representing the filter of the swimlane. Each swimlane can then be referenced as an input by other components with the name `<transform_name>.<swimlane_id>`."
+			description: "A table of lane identifiers to logical conditions representing the filter of the lane. Each lane can then be referenced as an input by other components with the name `<transform_name>.<lane_id>`."
```

Comment:
Should we rename `lane` to `route` here? Or some variant of that? I feel like route -> lane aren't a good combination. /cc @binarylogic.

Reply:
Using `route` here could be a bit confusing, since `route` in transform and `route` here have different meaning.

Reply:
`track` seams as a good alternative to `lane`. Has more of "controlled redirection" meaning.

Reply:
We could also call the component `router` and each "lane" a `route`. I'm content with `track` though.

Reply:
I don’t think it’s wrong to name both the transform and routes ‘route’. IMO, that is the most intuitive.

Reply:
Ok, then let's go with `route`, it's also the simplest option.